install.sh

```diff
@@ -235,6 +235,26 @@ End of functions from https://github.com/client9/shlib
 ------------------------------------------------------------------------
 EOF
 
+help() {
+  cat <<EOF
+Usage: $0
+
+Install:
+  $0
+
+Install to <path/to/dir>:
+  BIN_DIR=<path/to/dir> $0
+
+Install specified version <vX.Y.Z>:
+  DAGGER_VERSION=<vX.Y.Z> $0
+
+Install latest nightly build:
+  DAGGER_COMMIT=head $0
+Install specified nightly build <commit sha>:
```

Comment:
Can we rename `nightly` to `dev` ? That's what these are, since **every** commit to `main` produces one of these builds.